ktor-server/ktor-server-plugins/ktor-server-metrics/jvm/src/io/ktor/server/metrics/dropwizard/DropwizardMetrics.kt

```diff
@@ -105,3 +99,19 @@ public val DropwizardMetrics: ApplicationPlugin<DropwizardMetricsConfig> =
             }
         }
     }
+
+private fun PluginBuilder<DropwizardMetricsConfig>.callFinished(call: ApplicationCall) {
+    val routingMetrics = call.attributes.takeOrNull(routingMetricsKey)
+    val name = routingMetrics?.name ?: call.request.routeName
+    val status = call.response.status()?.value ?: 0
+    val statusMeter =
+        pluginConfig.registry.meter(name(pluginConfig.baseName, name, status.toString()))
+    statusMeter.mark()
+    routingMetrics?.context?.stop()
```

Comment:
It seems to me that in the case of missing `routingMetrics` timer measurement is going to be missing too.

Reply:
If routing didn't start then timer measurement wouldn't show anything useful. So I'm not sure but I think it's not critical

Reply:
But http call using method OPTIONS also have latency and it might be useful to measure.

I think that the problem that CORS plugin work around Routing plugin which is a bit weird. 

Seems like a low pri to me, but someone make think otherwise.

Reply:
CORS plugin replies to OPTIONS request before any route is matched. That's why there is no routing measure yet.